sql/testdata/builtin_function

```diff
@@ -495,3 +495,57 @@ query B
 SELECT experimental_uuid_v4() != experimental_uuid_v4(), length(experimental_uuid_v4())
 ----
 true 16
+
+query T
+SELECT GREATEST(4, 5, 7, 1, 2)
+----
+7
+
+query T
+SELECT LEAST(4, 5, 7, 1, 2)
+----
+1
+
+query T
+SELECT GREATEST(4, NULL, 7, 1, 2)
+----
+7
+
+query T
+SELECT GREATEST(NULL, NULL, 7, NULL, 2)
```

Comment:
Two more potential edge cases to test (I don't think there's a bug, just that these would be nice for more test coverage):
* Only the last value is non-null (so the second loop doesn't execute)
* Only one non-null value, which is not in the last position (so the second loop executes but never finds a non-null value)